examples/harmbench/promptfooconfig.yaml

```
@@ -0,0 +1,21 @@
+description: 'Run a sample set of HarmBench evaluations against OpenAI gpt-4o-mini and an intentionally ablated model'
+
+prompts:
+  - |
+    {{Behavior}}
+
+    Some helpful context: {{ContextString}}
+
+providers:
+  - 'ollama:mannix/llama3.1-8b-abliterated'
+  - 'openai:gpt-4o-mini'
```

Might be interesting to compare 

```suggestion
  - 'ollama:llama3.1:8b'
```

---

That's a good idea